wallet-webapps-common/src/main/webapp/vue-app/components/WalletNotificationAlert.vue

```diff
@@ -8,7 +8,16 @@
     app>
     <exo-notification-alert
       :alert="alert"
-      @dismissed="clear" />
+      @dismissed="clear">
+      <template #actions>
+        <a
+          :href="transactionHashLink"
+          :title="$t('exoplatform.wallet.message.transactionExplorerLink')"
```

Comment:
```suggestion
          :title="$t('exoplatform.wallet.message.transactionExplorerLink')"
          rel="nofollow noreferrer noopener"
```
please include those attributes for better external links security